internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -73,8 +76,7 @@ func resourceLogAnalyticsWorkspace() *pluginsdk.Resource {
 			"sku": {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
-				ForceNew: true,
-				Default:  string(operationalinsights.WorkspaceSkuNameEnumPerGB2018),
+				Computed: true,
 				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
we'll need to add a TODO here to add the new SKU in 4.x, remove computed, split the Create/Update and tell users to use `ignore_changes` here fwiw - since that's how this should be updated longer-term

Reply:
just handle it the same way it was handled for AKS

Reply:
@mozts2005 unfortunately that'd be a series of breaking changes which we can't ship until the next major version, so that's not possible in a minor release.

Reply:
I don't see why they would be braking changes you could just remap the old values to the new values.

Reply:
https://github.com/hashicorp/terraform-provider-azurerm/pull/15108/files#diff-ea42d8dfc931c56586c02b2273ca0e4e1deac5f45c9f19f1ae0b7722362dba34

Reply:
also, your changes do not allow of creating a new LAW with the cluster settings.

Reply:
my need to is to make so I can use azurerm_log_analytics_workspace and azurerm_log_analytics_cluster together in an Idempotent fashion from creation and in the case of an update.

Reply:
> 

This is by design, new LAWs can not be created with the cluster settings.